concrete/attributes/user_selector/controller.php

```diff
@@ -89,4 +89,16 @@ public function exportValue(\SimpleXMLElement $akn)
             $avn = $akn->addChild('value', $user->getUserID());
         }
     }
+
+    public function searchForm($list) {
+        $UserID = $this->request('value');
+        $list->filterByAttribute($this->attributeKey->getAttributeKeyHandle(), $UserID, '=');
+        return $list;
+    }
+
+    public function search() {
+        $form_selector = $this->app->make('helper/form/user_selector');
+        print $form_selector->selectUser($this->field('value'), $this->request('value'), false);
```

Comment:
You shouldn't really do this, you should have a search.php template. Otherwise to override the search field would mean you have to rewrite the controller.

Reply:
OK!